azerothcore/azerothcore-wotlk
# fix(Core/Spell): Handle ClearAllDebuffs

Cherry-pick part of https://github.com/TrinityCore/TrinityCore/commit/85c912c4048287931985c98186423d2bf51fdebb
Cherry-pick https://github.com/TrinityCore/TrinityCore/commit/8de7e12a6e16dd9e38deafe683e2165a26125410

## Changes Proposed:
-  Handle ClearAllDebuffs spell

## Tests Performed:

- Builds
- Tested in-game


## How to Test the Changes:

1.
```
.aura 3237
.aura 45677
.aura 38495 
.aura 44185 
.aura 31399
.aura 40075
```
2. .cast 34098
3. see it will remove the debuffs


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
